README.md

```diff
@@ -513,7 +513,20 @@ MessageDb support is provided in the samples and the `eqx` tool:
 - being able to supply `mdb` flag to `eqx dump`, e.g. `eqx dump -CU "Favorites-ab25cc9f24464d39939000aeb37ea11a" mdb -c "pgconnectionstring"`
 - being able to supply `mdb` flag to Web sample, e.g. `dotnet run --project samples/Web/ -- mdb -c "pgconnectionstring"`
 
-Equinox does not provide utilities for configuring or installing message-db. See [MessageDB's installation documentation](http://docs.eventide-project.org/user-guide/message-db/install.html).
+Equinox does not provide utilities for configuring or installing MessageDB. See [MessageDB's installation documentation](http://docs.eventide-project.org/user-guide/message-db/install.html).
+
+In addition to the default access strategy of reading the whole stream forwards in batches, the following access strategies are supported in MessageDb:
```

Comment:
will probably mention this above https://github.com/jet/equinox/blob/c9d689926e1a1a147febf8c3d9de2286a92e3696/README.md#currently-supported-data-stores together with a mention of it having otel wiring